### PR TITLE
Tech

### PR DESCRIPTION
**Pull request by davidtito1992**
## Summary

I add to the file 'index.html', below the code that represents the title, the code that represents a technology. We will also add to the 'application.scss' another '@ import' including a new 'tech.scss' file that contains the styles that are applied to the technologies.

## Trello Card 

https://trello.com/c/MLX3d19z/22-tecnologia

## Screenshots

captura de pantalla 2018-09-13 a la s 15 53 09

### CODE REVIEW

File: Maquetado/index.html [hunk 1]
```diff
@@ -14,6 +14,14 @@
     <p class='text'>Front-End Tools</p>
   </div>
 
+  <div class="tech">
+    <div class="content-tech">
+      <img class="img">
+      <h2 class="h2">ReacJS</h2>
+      <h3 class="h3">Release:2013</h3>
+      <p class="text">Reac is a declarative, component-based JavaScript library for building user interfaces</p>
```
Comment (CarlosHWolox) [inline Maquetado/index.html]: You should use more descriptive class names. Like tech, tech-title, tech-release-date, tech-text- tech-image

File: Maquetado/scss/tech.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
+$color: #b2db59;
+$urlImagen: "./assets/react_icon.svg";
+
+.tech {
+  border: 1px solid $color;
+  height: 300px;
+  border-radius: 10px;
+  margin:10px;
```
Comment (CarlosHWolox) [inline Maquetado/scss/tech.scss]: space

File: Maquetado/index.html [hunk 3]
```diff
@@ -14,6 +14,14 @@
     <p class='text'>Front-End Tools</p>
   </div>
 
+  <div class="tech">
+    <div class="content-tech">
```
Comment (CarlosHWolox) [inline Maquetado/index.html]: i think you can do this in just one div element

File: Maquetado/index.html [hunk 4]
```diff
@@ -14,6 +14,12 @@
     <p class='text'>Front-End Tools</p>
   </div>
 
+  <div class="tech">
+    <img class="tech-img" src="./assets/react_icon.svg">
+    <h2 class="tech-title">ReacJS</h2>
```
Comment (SKOLZ) [inline Maquetado/index.html]: It's "reactJS" haha

File: Maquetado/scss/tech.scss [hunk 5]
```diff
@@ -0,0 +1,51 @@
+$color: #b2db59;
```
Comment (SKOLZ) [inline Maquetado/scss/tech.scss]: This should be added to a colors.scss file inside a `variables` folder and import it in the application.scss at the begining so every file that comes after that one can use the colors declared

File: Maquetado/scss/tech.scss [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+
+.tech {
+  align-items: center;
+  background-color: $color;
+  border-radius: 10px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
```
Comment (SKOLZ) [inline Maquetado/scss/tech.scss]: Spacing between elements can be handled using margin bottom in every element and using justify content flex start. By doing space around a longer text in the description means less space beteen elements.

File: Maquetado/scss/tech.scss [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+  .tech-img {
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 70px;
+    width: 80px;
+  }
+
+  .tech-title {
+    margin-top: -40px;
```
Comment (SKOLZ) [inline Maquetado/scss/tech.scss]: Avoid negative margins at all costs. I think you dont need this, why are you using it?

File: Maquetado/scss/tech.scss [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+  }
+
+  .tech-release-date {
+    font-weight: bold;
+    margin-top: -50px;
```
Comment (SKOLZ) [inline Maquetado/scss/tech.scss]: Same here, not needed

File: Maquetado/scss/tech.scss [hunk 9]
```diff
@@ -0,0 +1,51 @@
...
+  }
+
+  .tech-text {
+    justify-self: flex-end;
+    font-family: sans-serif, 'Times New Roman', Times, serif;
+    text-align: center;
+    margin-bottom: 15px;
```
Comment (SKOLZ) [inline Maquetado/scss/tech.scss]: Use padding in the parent

File: Maquetado/scss/tech.scss [hunk 10]
```diff
@@ -0,0 +1,51 @@
...
+    margin-top: -40px;
```
Comment (SKOLZ) [inline Maquetado/scss/tech.scss]: Not needed

File: Maquetado/scss/tech.scss [hunk 11]
```diff
@@ -0,0 +1,51 @@
...
+  }
+
+  .tech-release-date {
+    font-weight: bold;
+    margin-top: -50px;
+  }
+
+  .tech-text {
+    justify-self: flex-end;
+    font-family: sans-serif, 'Times New Roman', Times, serif;
```
Comment (SKOLZ) [inline Maquetado/scss/tech.scss]: This is not necessary, use the imported google font.

File: Maquetado/scss/tech.scss [hunk 12]
```diff
@@ -0,0 +1,51 @@
...
+    margin-top: -40px;
+  }
+
+  .tech-release-date {
+    font-weight: bold;
+    margin-top: -50px;
+  }
+
+  .tech-text {
+    justify-self: flex-end;
```
Comment (SKOLZ) [inline Maquetado/scss/tech.scss]: You can remove this property since we'll align everything to the top

File: .gitignore [hunk 13]
```diff
@@ -1,3 +1,5 @@
 Maquetado/sass-cache
 Maquetado/application.css
 Maquetado/application.css.map
+Maquetado/.DS_STORE
+.DS_STORE
```
Comment (SKOLZ) [inline .gitignore]: Add empty line at eof

File: Maquetado/scss/tech.scss [hunk 14]
```diff
@@ -0,0 +1,48 @@
...
+    margin-bottom: 10px;
+  }
+
+  .tech-release-date {
+    font-weight: bold;
+    margin-bottom: 10px;
+  }
+
+  .tech-text {
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen,
```
Comment (SKOLZ) [inline Maquetado/scss/tech.scss]: This is not needed.

File: Maquetado/scss/tech.scss [hunk 15]
```diff
@@ -0,0 +1,48 @@
...
+    content: "";
+    left: -10px;
+    position: absolute;
+    right: 10px;
+    top: -10px;
+    z-index: -1;
+  }
+
+  .tech-img {
+    background-repeat: no-repeat;
```
Comment (SKOLZ) [inline Maquetado/scss/tech.scss]: there is no background here

File: Maquetado/scss/tech.scss [hunk 16]
```diff
@@ -0,0 +1,48 @@
...
+    background-size: cover;
```
Comment (SKOLZ) [inline Maquetado/scss/tech.scss]: same

File: Maquetado/scss/tech.scss [hunk 17]
```diff
@@ -0,0 +1,48 @@
...
+    height: 70px;
+    margin-bottom: 20px;
+    width: 80px;
```
Comment (SKOLZ) [inline Maquetado/scss/tech.scss]: I'd just specify the height here and let the width adapt

File: Maquetado/scss/tech.scss (hunk 14 above)
Comment (SKOLZ) [reply Maquetado/scss/tech.scss]: remove this line

File: Maquetado/scss/titles.scss [hunk 18]
```diff
@@ -1,16 +1,9 @@
-$urlImagen: "./assets/triangle.svg";
+$urlImagen: './assets/triangle.svg';
```
Comment (SKOLZ) [inline Maquetado/scss/titles.scss]: use english for variable names, and also use spinal- case. `$image-url` would be a good choice here.

**Pull request merged**